git-annex-remote-rclone

```diff
@@ -116,8 +116,8 @@ do_checkpresent() {
 	dest="$2"
 
 	res=0
-	check_result=$(rclone size --json "$dest" 2>/dev/null) || res=$?
-	printcmdresult "rclone size --json \"$dest\"" "$res" "$check_result"
+	check_result=$(rclone size ${RCLONE_FLAGS:+$RCLONE_FLAGS} --json "$dest" 2>/dev/null) || res=$?
```

Comment:
instead of repeating this, may be at the top of the file just have

```bash
: ${RCLONE_FLAGS:=}
```
so we just assign it to empty value if not defined, and then here and elsewhere

```suggestion
	check_result=$(rclone size ${RCLONE_FLAGS} --json "$dest" 2>/dev/null) || res=$?
```
?